src/service/device.js

```diff
@@ -733,6 +740,24 @@ var Device = GObject.registerClass({
         Gio.AppInfo.launch_default_for_uri_async(uri, null, null, null);
     }
 
+    openPathSelectFile(action, parameter) {
+        const path = parameter.unpack();
+        const uri = path.includes('://') ? path : `file://${path}`;
+
+        Gio.DBus.session.call(
+            'org.freedesktop.FileManager1',
+            '/org/freedesktop/FileManager1',
+            'org.freedesktop.FileManager1',
+            'ShowItems',
+            new GLib.Variant('(ass)', [[uri], 's']),
+            null,
+            Gio.DBusCallFlags.NONE,
+            -1,
+            null,
+            null
+        );
```

Comment:
looks like there should be an error to catch here

https://www.andyholmes.ca/articles/dbus-in-gjs.html#direct-calls

Logging the error is enough.